keras_ocr/pipeline.py

```diff
@@ -28,7 +28,7 @@ def recognize(self, images, detection_kwargs=None, recognition_kwargs=None):
         """Run the pipeline on a single image.
```

Comment:
Could you also update this description to reflect that we now accept multiple images?

Reply:
Done